packages/suite/src/views/wallet/transactions/TransactionList/WalletTransactionList.tsx

```diff
@@ -0,0 +1,91 @@
+import { ReactNode, useState } from 'react';
+import { useDispatch } from 'react-redux';
+
+import styled from 'styled-components';
+
+import { areNetworkTransactionsUnreliable } from '@suite-common/wallet-config';
+import { Card, Column, Link } from '@trezor/components';
+import { typography } from '@trezor/theme';
+
+import { goto } from 'src/actions/suite/routerActions';
+import { Translation } from 'src/components/suite';
+import { Account, WalletAccountTransaction } from 'src/types/wallet';
+
+import { TransactionList } from './TransactionList';
+import { useVisibleTransactions } from './useFetchTransactions';
+
+const NoResults = styled.div`
+    text-align: center;
+    color: ${({ theme }) => theme.legacy.TYPE_DARK_GREY};
+    ${typography.hint}
+`;
+
+export const NoVisibleTransactions = () => {
+    const dispatch = useDispatch();
+
+    return (
+        <Card>
+            <NoResults>
+                <Column alignItems="center">
+                    <Translation
+                        id="TR_NO_VISIBLE_TRANSACTIONS"
+                        values={{
+                            tabLink: (chunks: ReactNode[]) => (
+                                <Link
+                                    onClick={event => {
+                                        event.preventDefault();
+                                        dispatch(
+                                            goto('wallet-tokens-hidden', { preserveParams: true }),
+                                        );
+                                    }}
+                                >
+                                    {chunks}
+                                </Link>
+                            ),
+                        }}
+                    />
+                </Column>
+            </NoResults>
```

Comment:
I think something like this should work, but please test it. I think `NoResult` component is not necessary.

```suggestion
                <Column alignItems="center">
                <Text typographyStyle="hint" variant="tertiary">
                    <Translation
                        id="TR_NO_VISIBLE_TRANSACTIONS"
                        values={{
                            tabLink: (chunks: ReactNode[]) => (
                                <Link
                                    onClick={event => {
                                        event.preventDefault();
                                        dispatch(
                                            goto('wallet-tokens-hidden', { preserveParams: true }),
                                        );
                                    }}
                                >
                                    {chunks}
                                </Link>
                            ),
                        }}
                    />
                    <Text>
                </Column>
```